### PR TITLE
AAP-18228: Handle user's trial expiry

### PR DESCRIPTION
**Pull request by romartin**
Hello, 

See https://issues.redhat.com/browse/AAP-18228

This PR is **related to** https://github.com/ansible/ansible-wisdom-service/pull/761 

Thanks in advance!
Roger

### CODE REVIEW

File: src/features/lightspeed/handle_api_error.ts [hunk 1]
```diff
@@ -43,6 +43,12 @@ export function retrieve_error(err: AxiosError): string {
           "permission_denied__org_not_ready_because_wca_not_configured"
       ) {
         return `Contact your administrator to configure IBM watsonx Code Assistant model settings for your organization.`;
+      } else if (
+        responseErrorData &&
+        responseErrorData.hasOwnProperty("code") &&
+        responseErrorData.code === "permission_denied__user_trial_expired"
+      ) {
+        return `Trial for Ansible Lightspeed user expired. Contact your Red Hat Organization's administrator for more information on how to fix this.`;
```
Comment (robinbobbitt) [inline src/features/lightspeed/handle_api_error.ts:51]: Marty needs to provide the actual message here.

**Review by ganeshrn — COMMENTED** [review]
Please add unit test [here](https://github.com/ansible/vscode-ansible/blob/main/test/units/lightspeed/handle_api_error.test.ts) to cover this case.
You can run the unit test using command `yarn run unit-tests`

**Comment by hasys** [comment]
Hi @romartin, @ganeshrn,

Unit tests and final Trial expiry message is added in the new PR: https://github.com/ansible/vscode-ansible/pull/1037 this PR can be closed.

Thank you!

**Comment by ganeshrn** [comment]
PR superseded by https://github.com/ansible/vscode-ansible/pull/1037

**Pull request closed without merging**